.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: objective-c  #Travis only recognizes Obj-C, Xcode will handle the fact that this is Swift
+
+# xcode and sdk versions
+osx_image: xcode7
+xcode_sdk: iphonesimulator9.0
+xcode_project: "APP/HMC Word Scroll/HMC Word Scroll.xcodeproj"
+xcode_scheme: "HMC Word Scroll"
+
+# Update any of our needed tools
+before_install:
```

Comment:
Probably unnecessary, but good practice for how to update things using home-brew